### PR TITLE
trojan 1.16.0 (new formula)

### PR DESCRIPTION
**Pull request by kidonng**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/trojan.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          "verify": false,
+          "verify_hostname": false
+        }
+      }
+    EOS
+    server = fork { exec bin/"trojan", "-c", testpath/"server.json" }
+    client = fork { exec bin/"trojan", "-c", testpath/"client.json" }
+    sleep 3
+    begin
+      system "curl", "--socks5", "127.0.0.1:#{local_port}", "github.com"
```
Comment (SMillerDev) [inline Formula/trojan.rb:67]: You need to check the output here.

File: Formula/trojan.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    ensure
+      Process.kill 9, server
+      Process.wait server
+      Process.kill 9, client
+      Process.wait client
+    end
```
Comment (SMillerDev) [inline Formula/trojan.rb:73]: This isn't needed anymore. Brew kills processes in its own.

Comment (kidonng) [reply Formula/trojan.rb]: This is taken from an old formula, thanks for the tip.

**Comment by carlocab** [comment]
```
CMake Error at cmake_install.cmake:59 (file):
  file cannot create directory:
  /etc/opt/homebrew/Cellar/trojan/1.16.0/trojan.  Maybe need administrative
  privileges.
```
Looks like a bug in their build scripts.

**Pull request closed without merging**